StudentEmployeeInformation/admin.py

```diff
@@ -1,5 +1,13 @@
 from django.contrib import admin
-from .models import Students
+from .models import Student, UniveristyClass, ISStaffMember, Season, Semester, ClassSection, ResearchSection, YearInProgram, StudentEmployeePosition
 
 # Register your models here.
-admin.site.register(Students)
\ No newline at end of file
+admin.site.register(Student)
```

Comment:
These lines are necessary to get the models to show up on the Django admin thing.